apps/src/sites/code.org/pages/public/learn/local.js

```diff
@@ -52,6 +57,21 @@ $(function() {
     });
 });
 
+function checkOffset() {
```

Comment:
Copying some documentation to here as well.
Also consider adding some inline comments below explaining the maths we are doing.
```suggestion
// Prevent the map from covering the footer
function checkOffset() {
```